XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -76,23 +76,23 @@
           "Docs": [
             {
               "Title": "AppBarButton - API",
-              "Uri": "http://msdn.microsoft.com/library/windows/apps/windows.ui.xaml.controls.appbarbutton.aspx"
+              "Uri": "https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.appbarbutton"
```

Comment:
Adding the "en-us" designation will break localization for folks outside of the en-us locale. 

Leaving "en-us" out will auto-resolve to the correct locale for the user. 

For example: https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.appbarbutton

Please update the links in this file to remove the "en-us" locale.

Reply:
I closed it because I noticed the en-us issue, and also thought I missed some links (was looking at an old version by mistake).

I removed the VS 2017 link with the intention to just use: https://visualstudio.microsoft.com/vs/ - which pushes 2019 as the version of choice now, but forgot that too. 

Then I made the changes, got distracted, and forgot to do another PR. A+ work. I'll blame trying to use github's built-in editor...